web/client-api/types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@deephaven/jsapi-types",
-  "version": "1.0.0-dev0.31.0",
+  "version": "0.0.0-dummy",
```

Comment:
explicit dummy version to be clear that developers need not update this, it will automatically be updated in the gradle+docker wiring